VL.ImGui.Stride/shaders/ImGuiShader_DrawFX.sdsl

```diff
@@ -0,0 +1,26 @@
+﻿shader ImGuiShader_DrawFX : ColorUtilityTemp, ShaderBase 
+{
+
+    matrix proj;
+    stage Texture2D tex;
+
+    SamplerState TexSampler {
+        Filter = ANISOTROPIC;
+        AddressU = Wrap;
+        AddressV = Wrap;
+        MaxAnisotropy = 16;
+    };
+
+    stream float2 pos : POSITION;
+    stream float2 uv : TEXCOORD0;
+    stream float4 col : COLOR;
+
+    override stage void VSMain() {
+        streams.ShadingPosition = mul(proj, float4(streams.pos, 0.0, 1.0f)) + float4(-1.0f, 1.0f, 0.0f, 0.0f);
+    }
+
+    override stage void PSMain() {
+        streams.ColorTarget = SRgbToLinearPrecise(streams.col) * tex.Sample(TexSampler, streams.uv);
```

Comment:
I think SRgbToLinearPrecise can go away once we identify all places where colors get fed to ImGui and convert them on the CPU side.